README.md

```diff
@@ -0,0 +1,38 @@
+# GitHub Meta project (.github)
+
+This repository manages community files, which are relevant for all our projects.
+
+It provides a default for specific files and can be overwritten by each project. See the [GitHub Documentation](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/creating-a-default-community-health-file) regarding community Health files for further information.
+
+## Contents
+
+### Community Health Files
+
+The following files will be used as a fallback for all the projects within the same organization.
+
+- [Code of Conduct](CODE_OF_CONDUCT.md)
+- [Contributing Guidelines](CONTRIBUTING.md)
+- [Security Policy](SECURITY.md)
+- [ISSUE_TEMPLATES](ISSUE_TEMPLATES/) for bug reports and freature requests
+- `.github/*.yml` for bot configurations like [Probot](https://probot.github.io/docs/best-practices/#store-configuration-in-the-repository)
```

Comment:
Is this for configuration within this repo or will it be inherited by all other repos?